index.js

```diff
@@ -3983,6 +3983,31 @@
   S.splitOn =
   def('splitOn', {}, [$.String, $.String, $.Array($.String)], splitOn);
 
+  //# splitOnRegex :: RegExp -> String -> Array String
+  //.
+  //. Type-checked version of String#split that accepts RegEx as a separator.
```

Comment:
I would like to make `String#split` itself a link to the MDN documentation:

    Type-checked version of the regular expression mode of [`String#split`][].

We'd need to include a reference for ``[`String#split`]`` at the bottom of the document.